terminal_colors/terminal_colors.py

```diff
@@ -554,10 +560,22 @@ def _terminal():
         terminal class
     """
     curses.setupterm()
-    num_colors = curses.tigetnum('colors')
-    return {16:term16, 88:term88, 256:term256}.get(num_colors, term16)
```

Comment:
I changed this inline "constant" dict to a global constant variable `TERM_TABLE` so that it could be referenced in the OptionParser to constrain the possible values passed to `--force-term`. Your code style convention does not use ALL_CAPS for constants but I couldn't bring myself not to. :)